committee-steering/governance/README.md

```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the steering committee to gain approval.
```

Comment:
Wouldn't working with the steering to gain approval be a latter step in the process (after circulating changes with the sig and/or community)?

Reply:
I was structuring this as an OARP type thing -- Owners, Approvers, Reviewers and Participants.  I'll rework it so that it is viewed as more of a sequential process.